src/database/series.ts

```diff
@@ -45,6 +45,7 @@ function SeriesVolume(seriesId: string, volume: string, startYear: number, title
   if (Number(volume) > 1) {
     this.titleWithVolume += ' Vol. ' + volume;
   }
+  this.searchTitleWithVolume = title += ' Vol. ' + volume;
```

Comment:
What does the `+=` here do?

Reply:
Copy paste error and  not for this branch